modules/persistent-storage-csi-snapshots-create.adoc

```diff
@@ -91,22 +98,23 @@ spec:
 ----
 <1> The volumeSnapshotContentName parameter is required for pre-provisioned snapshots.
 
-. Create the object you saved in the previous step:
+. Create the object you saved in the previous step by running the following command:
 +
 ----
 $ oc create -f volumesnapshot.yaml
 ----
 
```

Comment:
the manifest name for creating volumesnapshot should be `volumesnapshot-manual.yaml`

Reply:
Good catch. I also updated the manifest name for creating dynamic volumesnapshot to `volumesnapshot-dynamic.yaml` above.